config/locales/en/pages.yml

```diff
@@ -1,5 +1,31 @@
 en:
   pages:
+    "404":
+      title: This page cannot be found
+      body_html: |
+        <p>You can <a href="/">start a new prison visit request</a> or book
+        your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "500":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available right now, but
+        we're working hard to get things up and running again.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "503":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available as we're carrying
+        out some planned maintenance. You can try again later or book your
+        visit by phone instead.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
```

Comment:
I'm not really sold... For 1, I can't see a scenario where these paragraphs were split / joined without reference to the copy, but I can foresee plenty of scenarios where copy changes dictate adding or removing paragraphs. In terms of keeping changes. 2 and 3 are valid, but I don't think in either case that coupling paragraphs between the copy file and the templates is a good solution to either problem.

And compare with the terms and conditions page: https://github.com/ministryofjustice/prison-visits-2/blob/master/config/locales/en/pages.yml#L10. There's no way it makes sense to split that page up, while allowing markdown to be written would be more user friendly and safer.